index.js

```diff
@@ -28,52 +26,45 @@ var encodingMethods = {
  * @api public
  */
 
-module.exports = (options) => {
-  options = options || {}
-
-  var filter = options.filter || compressible
-
-  var threshold = !options.threshold ? 1024
-    : typeof options.threshold === 'number' ? options.threshold
-    : typeof options.threshold === 'string' ? bytes(options.threshold)
-    : 1024
+module.exports = (options = {}) => {
+  let { filter = compressible, threshold = 1024 } = options;
```

Comment:
I did believe that people who do care about semicolons are first to have an `eslint` config in the project... Removed! ;-)